src/renderer/components/cluster-settings/components/cluster-home-dir-setting.tsx

```diff
@@ -33,7 +33,6 @@ export class ClusterHomeDirSetting extends React.Component<Props> {
     return (
       <>
         <SubTitle title="Working Directory"/>
-        <p>Terminal working directory.</p>
```

Comment:
I think you should rename the subtitle then since working directory is an ambiguous phrase here.

Reply:
Yup, it is good.